pkg/compact/planner_test.go

```diff
@@ -800,7 +800,7 @@ func TestLargeTotalIndexSizeFilter_Plan(t *testing.T) {
 			t.Run("from meta", func(t *testing.T) {
 				obj := bkt.Objects()
 				for o := range obj {
-					delete(obj, o)
+					bkt.Delete(context.Background(), o)
```

Comment:
Let's check the error here.
```suggestion
					testutil.Ok(t, bkt.Delete(context.Background(), o))
```